### DISCUSSION TITLE
2024-10-22-r1

### DISCUSSION THREAD

**Discussion by Zoey2936** [ZoeyVid/NPMplus#1180] [Announcements]
# Breaking changes in last release, please read the last changelog here: https://github.com/ZoeyVid/NPMplus/releases/tag/2024-10-21-r1

## What's Changed
- fix launch of nginx if certbot-ocsp-fetcher.sh fails
- merge upstream (no real changes, formatting of certbot-dns-plugins.json and add some upstreams envs to compose.yaml)
- dep updates

### How to update
- Read the changes above
- Pull the zoeyvid/npmplus:latest image
- apply possible changes that maybe effect you from above to your compose.yaml/NPMplus
- redeploy the compose stack
- report any issues you find

**Full Changelog**: https://github.com/ZoeyVid/NPMplus/compare/2024-10-05-r1...2024-10-21-r1

**Full Changelog**: https://github.com/ZoeyVid/NPMplus/compare/2024-10-21-r1...2024-10-22-r1

<hr /><em>This discussion was created from the release <a href='https://github.com/ZoeyVid/NPMplus/releases/tag/2024-10-22-r1'>2024-10-22-r1</a>.</em>

**Comment by Krazy998** [comment]
I had a strange issue. I havnt updated for a couple of months (probably 4 or so). Anyway after updating to latest I could see the admin login page, but failed to login as node wouldnt start for some reason. (everything else worked). 

I then ended up going to zoeyvid/npmplus:419 then to latest and it worked fine. So maybe some migration step was missing.

Just thought I would note.

**Reply by Zoey2936** [reply]
Thanks, was there anything insode the log?

**Reply by Krazy998** [reply]
hey @Zoey2936, docker logs for the container didnt show anything around errors. It just looked like node didnt start for some reason. 

Unfortunately I dont have any logs to share from that time :(. 

EDIT: I noticed another topic on the matter(https://github.com/ZoeyVid/NPMplus/discussions/1184), when logging in i can see same in the browser debug, 502 getting token. Thats when I noticed node process not started. So looks like you already narrowed this problem down :)

**Reply by Zoey2936** [reply]
All good, the issue was found here: #1184 and fixed in develop
